core/geometry/src/geometry3d/Matrix3d.ts

```diff
@@ -2098,25 +2098,28 @@ export class Matrix3d implements BeJSONFunctions {
     return result;
   }
   /**
-   * Multiply `this` Matrix3d (considered as a Transform with 0 translation) times `other` Transform.
+   * Multiply `this` Matrix3d (considered to be a Transform with 0 `origin`) times `other` Transform.
+   * **Note:** If `other = [B   b]`, then we promote `this` matrix to be a Transform [A   0].
+   * Then `this * other` is defined as [A*B   Aa]. That's because we create a 4x4 matrix for each Transform
+   * with the 3x3 `matrix` and `origin` as upper 3x4 part of a 4x4 matrix and 0001 as the final row. Then we
+   * multiply those two 4x4 matrixes:
    * ```
    * equation
    * \begin{matrix}
-   * \text{This matrix }\bold{A}\text{ promoted to block transform} & \blockTransform{A}{0} \\
-   * \text{other transform with matrix part }\bold{B}\text{ and translation }\bold{b} & \blockTransform{B}{b}\\
+   * \text{`this` matrix }\bold{A}\text{ promoted to block Transform} & \blockTransform{A}{0} \\
+   * \text{`other` Transform with `matrix` part }\bold{B}\text{ and `origin` part }\bold{b} & \blockTransform{B}{b}\\
```

Comment:
Does a single-quoted word in LaTeX display with special formatting like in a non-equation comment? If so, this is fine. If not, remove the quotes to simplify.

Reply:
will remove in next PR